components/presets.js

```diff
@@ -1,42 +1,78 @@
 export default [
   {
-    name: 'light',
-    colors: {
-      text: '#282230',
-      background: '#f1f1f1',
-      primary: '#01796f',
-      primary_dark: '#016159',
-      secondary: '#562931',
+    name: 'pure',
+    light: {
+      textColor: '#818d9f',
+      backgroundColor: '#f1f1f1',
+      button: {
+        primary: {
+          backgroundColor: 'var(--theme-colors-primary)',
+          color: 'var(--theme-text-color)',
+        }
+      },
+      colors: {
+        primary: '#fff',
+        secondary: '#562931'
+      },
+    },
+    dark: {
+      textColor: '#fff',
+      backgroundColor: '#121212',
+      button: {
+        primary: {
+          backgroundColor: '#222',
+          color: '#f1f1f1',
+        }
+      },
     },
   },
   {
-    name: 'dark',
-    colors: {
-      text: '#f1f1f1',
-      background: '#27323a',
-      primary: '#01978b',
-      primary_dark: '#00887c',
-      secondary: '#fe8690',
+    name: 'midnight',
+    light: {
+      textColor: '#f1f1f1',
+      backgroundColor: '#1e272e',
+      button: {
+        primary: {
+          backgroundColor: 'var(--theme-colors-primary)',
+          color: 'var(--theme-text-color)',
+        }
+      },
+      colors: {
+        primary: '#4b9eff',
+        secondary: '#fe8690',
+        text: '#f1f1f1',
+      },
+    },
+    dark: {
+      textColor: '#888',
+      backgroundColor: '#0d1215',
+      colors: {
+        primary: '#16242d',
+      },
     },
   },
   {
     name: 'forest',
-    colors: {
-      text: '#f9f2cf',
-      background: '#3b6c4c',
-      primary: '#efdc7e',
-      primary_dark: '#e4d589',
-      secondary: '#4a875f',
+    light: {
+      textColor: '#f9f2cf',
+      backgroundColor: '#3b6c4c',
+      button: {
+        primary: {
+          backgroundColor: 'var(--theme-colors-primary)',
+          color: 'var(--theme-text-color)',
```

Comment:
this was just showcase referencing other declared vars in the config so you can keep things controlled/DRY.

```css
backgroundColor: 'var(--theme-colors-primary)',
```

can also be ref'd this way
```css
backgroundColor: 'var(--theme-forest-colors-primary)',
```

Reply:
This was something I noticed that would break with the new prefix prop, I think if we followed what (there was a project built on emotion that was pretty popular in React before Chakra and tailwinds blew up, but the name is escaping me) where you can specify the color's key as a string, or the tentative kebab-case. So in this case instead of using the CSS var, we could use `"primary"` and do the magic behind the scenes to create that token.

Reply:
You could introduce a `prefix` macro that will be expanded during theme config processing inside `setCSS`. I just pushed a new commit to this PR with that change.